seqr/views/apis/data_manager_api_tests.py

```diff
@@ -687,14 +719,11 @@ def mock_write(content):
                 response_json = response.json()
                 self.assertDictEqual(response_json, {'info': info, 'warnings': warnings, 'sampleGuids': [mock.ANY], 'fileName': file_name})
                 deleted_count = params.get('deleted_count', params['initial_model_count'])
-                mock_logger.info.assert_has_calls(
-                    [mock.call(info_log, self.data_manager_user) for info_log in info] + [
-                        mock.call(f'delete {deleted_count} {model_cls.__name__}s', self.data_manager_user, db_update={
-                            'dbEntity': model_cls.__name__, 'numEntities': deleted_count, 'parentEntityIds': mock.ANY, 'updateType': 'bulk_delete',
-                        }),
-                    ], any_order=True
+                mock_logger.info.assert_has_calls([mock.call(info_log, self.data_manager_user) for info_log in info])
+                mock_model_logger.info.assert_called_with(
+                    f'delete {model_cls.__name__}s', self.data_manager_user,
+                    db_update={'dbEntity': model_cls.__name__, 'numEntities': deleted_count, 'updateType': 'bulk_delete'}
                 )
-                self.assertTrue(RNA_SAMPLE_GUID in mock_logger.info.call_args_list[1].kwargs['db_update']['parentEntityIds'])
```

Comment:
this should be added back

Reply:
The `RNA_SAMPLE_GUID` has been added to the `parentEntityIds` field of the predicted data.